mkosi/backend.py

```diff
@@ -627,6 +627,14 @@ def nspawn_rlimit_params() -> Sequence[str]:
     ]
 
 
+def nspawn_executable() -> str:
+    return os.getenv("MKOSI_NSPAWN_EXECUTABLE", "systemd-nspawn")
```

Comment:
should this env var be documented somewhere?

Reply:
Probably not, it's mostly for me anyway

Reply:
if it's included, we should document it - others might have the same use case (I might from time to time, and am very likely to forget)

Reply:
Added a snippet to the docs

Reply:
'Guess it would make sense to add this as a separate section in the man page. I always forget about the `TMPDIR` env variable, as that is only mentioned once somewhere inline.